js/src/components/google-ads-account-card/connect-ads/index.js

```diff
@@ -84,7 +84,7 @@ const ConnectAds = ( props ) => {
 			<Section.Card.Body>
 				<Subsection.Title>
 					{ __(
-						'Connect to an existing account',
+						'Or, connect to an existing account',
```

Comment:
Not sure why this wording was changed? This wording is only displayed if it's not yet connected to an existing account. The original looks more appropriate.

![1](https://github.com/user-attachments/assets/65b33934-05f3-42e1-8010-d5f1831415b5)

Reply:
Good catch. This was applied to the wrong component in addition to the `ConnectedGoogleComboAccountCard` component in [0f0aa16](https://github.com/woocommerce/google-listings-and-ads/pull/2660/commits/0f0aa16b8cd7f7f4cfe87bfafc36437900b92933). Reverted in [df93210](https://github.com/woocommerce/google-listings-and-ads/pull/2660/commits/df93210dfebcad82fb82de5286c44e1714be22da).